### DISCUSSION TITLE
SRS Python

### DISCUSSION THREAD

**Discussion by carlo-gli** [srsran/zynq_timestamping#67] [General]
Pls how do locate the SRS python_tools folder, 

what exactly is SRS , is it actually the srsran installaction.

I am new and trying to build an LTE Radio with ANTSDR . i was directed to this page unfortunately i am locked on getting into the SRS python_tools Directory.

**Comment by ofontbach** [comment]
Hey, those are part of the zynq timestamping solution. SRS is the (perhaps not very original) name of the python framework that the solution utilizes. If you follow the provided [build instructions](https://srsran.github.io/zynq_timestamping/bitstream/antsdr_project.html) those will be installed for you.
Cheers
